src/pages/Defi/views/StakingVaults.tsx

```diff
@@ -12,9 +16,48 @@ const DefiHeader = () => {
   )
 }
 
+const FoxFarmCTA = () => {
+  const ethLogoSrc = 'https://assets.coincap.io/assets/icons/256/eth.png'
+  const foxLogoSrc = 'https://assets.coincap.io/assets/icons/256/fox.png'
+  return (
+    <Card variant='outline' my={1}>
+      <Stack
+        direction={{ base: 'column', xl: 'row' }}
+        alignItems='center'
+        justifyContent='space-between'
+        px={10}
+        py={4}
+      >
+        <Flex alignItems='center'>
```

Comment:
```suggestion
        <Flex alignItems='center' mb={{ base: 2, sm: 0 }}>
```

Adjust the space between the button and the text on mobile, but as the product team didn't provide the button mockup on mobile, it could change later